src/css/main.scss

```diff
@@ -96,59 +124,70 @@
 }
 
 @include media-breakpoint-down(xs) {
-  .h-h1 {
+  // .h-h1,
+  .c-title,
+  .c-cover__title {
     padding-bottom: 2rem;
     font-size: 1.6rem;
   }
 
-  .h-txt24 {
+  // .h-txt24,
+  .c-menu--mobile__link,
+  .c-lang--mobile__item,
+  .c-section__header,
+  .c-quote__header,
+  .c-list-item__inner {
     font-size: 1.2rem;
   }
 
-  .h-txt16 {
+  // .h-txt16,
+  .c-section__text,
+  .c-quote__author {
```

Comment:
Not sure, but maybe we should also move `@include media-breakpoint-down(xs) {` to separate files?

Benefit:
* everything in one place
* separation of concern 

Example:

Before:

```scss
.c-section {
  &__header {
    margin: 2rem 0;
    font-size: 1.6rem;
    line-height: 100%;
    text-transform: uppercase;
    letter-spacing: 0.1em;
  }

  &__text {
    margin-bottom: 1.6rem;
    font-size: 1rem;
    letter-spacing: 0.05em;
  }

  &--cover {
    color: $col-white;
    text-shadow: 0 0 10px #000;
  }
}
```

After:

```scss
.c-section {
  &__header {
    margin: 2rem 0;
    font-size: 1.6rem;
    line-height: 100%;
    text-transform: uppercase;
    letter-spacing: 0.1em;
  }

  &__text {
    margin-bottom: 1.6rem;
    font-size: 1rem;
    letter-spacing: 0.05em;
  }

  &--cover {
    color: $col-white;
    text-shadow: 0 0 10px #000;
  }

  @include media-breakpoint-down(xs) {
    &__header {
      font-size: 1.2rem;
      margin: 1.2rem 0;
    }

    &__text {
      font-size: 0.8rem;
      margin-bottom: 1rem;
    }
  }
}
```

- [ ] merge it